subworkflows/nf-core/variantcalling/svdb/main.nf

```diff
@@ -0,0 +1,32 @@
+include { TABIX_BGZIPTABIX as TABIX_BGZIP_TIDDIT_SV } from '../../../../modules/nf-core/modules/tabix/bgziptabix/main'
```

Comment:
I don't see an obvious difference now between this one the tiddit/main.nf. Don't think this file is needed or used?

Reply:
Oh yes, good point, will delete!